wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -905,60 +892,6 @@ private void checkRestoredWalletEncryptionDialog() {
         }
     }
 
-    @Override
-    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMenuInfo menuInfo) {
-        super.onCreateContextMenu(menu, v, menuInfo);
-        MenuInflater inflater = getMenuInflater();
-        if (v == viewFakeForSafetySubmenu) {
-            inflater.inflate(R.menu.wallet_safety_options, menu);
-
-            final String externalStorageState = Environment.getExternalStorageState();
-
-            menu.findItem(R.id.wallet_options_restore_wallet).setEnabled(
-                    Environment.MEDIA_MOUNTED.equals(externalStorageState) || Environment.MEDIA_MOUNTED_READ_ONLY.equals(externalStorageState));
-            menu.findItem(R.id.wallet_options_backup_wallet).setEnabled(Environment.MEDIA_MOUNTED.equals(externalStorageState));
-            menu.findItem(R.id.wallet_options_encrypt_keys).setTitle(
-                    wallet.isEncrypted() ? R.string.wallet_options_encrypt_keys_change : R.string.wallet_options_encrypt_keys_set);
-
-            boolean showFingerprintOption = fingerprintHelper != null && !fingerprintHelper.isFingerprintEnabled();
-            menu.findItem(R.id.wallet_options_enable_fingerprint).setVisible(showFingerprintOption);
-        }
-    }
-
-    @Override
-    public boolean onContextItemSelected(MenuItem item) {
-        switch (item.getItemId()) {
-            case R.id.wallet_options_safety:
-                HelpDialogFragment.page(getSupportFragmentManager(), R.string.help_safety);
-                return true;
-
-            case R.id.wallet_options_backup_wallet:
-                handleBackupWallet();
-                return true;
-
-            case R.id.wallet_options_restore_wallet:
-                handleRestoreWallet();
-                return true;
-
-            case R.id.wallet_options_encrypt_keys:
-                handleEncryptKeys();
-                return true;
-            case R.id.wallet_options_backup_wallet_to_seed:
-                handleBackupWalletToSeed();
-                return true;
-
-            case R.id.wallet_options_restore_wallet_from_seed:
-                handleRestoreWalletFromSeed();
-                return true;
-
-            case R.id.wallet_options_enable_fingerprint:
-                enableFingerprint();
-                return true;
-        }
-
-        return super.onContextItemSelected(item);
-    }
-
```

Comment:
Let's not remove any of the stuff from the Navigation Drawer menu.  I think this should be a separate story after we have moved everything over to the the new Menu system.

Reply:
In the past having this old navigation menu was very useful during the testing and development phase because it has a few working functions not yet available in the redesigned menus.

Reply:
The code should stay active rather than being commented out.

This will also reduce merge conflicts with the Change PIN story that should be come a PR very soon.

Reply:
reverted